src/test/java/com/github/romankh3/image/comparison/ImageComparisonUnitTest.java

```diff
@@ -46,6 +46,20 @@ public void testCorrectWorkingExpectedActual() throws IOException {
         assertImagesEqual(expectedResultImage, comparisonResult.getResult());
     }
 
+    @Test
+    public void testImageWithAdditional1pxLine() throws IOException {
+        //given
+        ImageComparison imageComparison = new ImageComparison("onePxLineNo.png", "onePxLineYes.png");
+        BufferedImage expectedImage = readImageFromResources("onePxLineDiff.png");
```

Comment:
Please, update test name and image names as it was before:
images `expected`, `actual` and `result`. Names of the images contain number of the issue. For example: `actual#17.png`.

The name of the test also contains an issue number.
For example: `testIssue134()`